docs/reference/configuration/data_types.rst

```diff
@@ -532,8 +532,7 @@ Pecularites of EnKF heritage include:
 
 1. The `FIELD` implementation can behave both as a dynamic quantity, i.e.
```

Comment:
Also a reference to dynamic here.

Reply:
I guess that FIELD here refers to fields in general while the next sentence states what ERT supports. 
If we remove the dynamic reference here I guess it makes sense to remove the whole paragraph.

Reply:
Yes, agree that makes sense :+1: